.github/workflows/activerecord_appraisals.yml

```diff
@@ -0,0 +1,37 @@
+name: ActiveRecord Appraisals
+
+on:
+  schedule: [ cron: '0 0 * * *' ]
+  pull_request:
+
+jobs:
+  appraise:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        ruby:
+          - '3.1'
+          - '3.2'
+          - '3.3'
+          - '3.4'
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+        bundler-cache: true
+
+    - name: Resolve appraisals dependencies
+      run: bundle exec appraisal install 
```

Comment:
I wonder if we can [cache](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/caching-dependencies-to-speed-up-workflows) this step?